packages/node_modules/nav-frontend-stegindikator/src/stegindikator.tsx

```diff
@@ -0,0 +1,211 @@
+import * as React from 'react';
+import * as cn from 'classnames';
+
+import { omit } from 'nav-frontend-js-utils';
+
+import 'nav-frontend-stegindikator-style';
+
+const cls = (state) => cn('stegindikator', {
+    'stegindikator--kompakt': state.kompakt
+});
+
+const stegCls = (props) => cn('stegindikator__steg', {
+    'stegindikator__steg--labelled': props.visLabel
+});
+
+const innerCls = (props) => cn('stegindikator__steg-inner', {
+    'stegindikator__steg-inner--aktiv': props.aktiv,
+    'stegindikator__steg-inner--ferdig': props.ferdig,
+    'stegindikator__steg-inner--disabled': props.disabled
+});
+
+export interface StegindikatorStegProps {
+    label: string;
+    index?: number;
+    visLabel?: boolean;
+    onClick?: () => void;
```

Comment:
Hvis `onClick` ikke sender ut noe data så greier jeg ikke helt å se behovet for den. Men basert på bruken nedenfor kan vi forvente at `event` kommer med her, så der bør reflekteres i typen.

Reply:
Fikset 👍